### PR TITLE
i18n(zh-cn): Translate images.mdx

### PR DESCRIPTION
**Pull request by liruifengv**
#### What kind of changes does this PR include?

- Translated content

#### Description

- Translate images.mdx
- https://github.com/withastro/docs/pull/3739

### CODE REVIEW

**Comment by sarah11918** [comment]
So excited for this @liruifengv !

Heads up that I made another commit with some changes to three of the `migrate-to/from-` guides: `https://github.com/withastro/docs/pull/3739/commits/58feb02f562be4bd0f113617b287ec25614079d7`

Then, there were some small conflict fixes in the following commits.

**Comment by liruifengv** [comment]
> So excited for this @liruifengv !↳
> 
> Heads up that I made another commit with some changes to three of the `migrate-to/from-` guides: `https://github.com/withastro/docs/pull/3739/commits/58feb02f562be4bd0f113617b287ec25614079d7`↳
> 
> Then, there were some small conflict fixes in the following commits.↳

Done!

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-gatsby.mdx [hunk 1]
```diff
@@ -320,41 +318,33 @@ Gatsby 中使用 `gatsby-browser.js` 中的 CSS 导入实现全局样式。在 A
...
 ### Gatsby 图像插件转换为 Astro
 
-:::note
-Astro v3.0 将包含一个新的 `astro:assets` 模块，并将弃用现有的 `@astrojs/image` 集成。
-
-目前已经可以通过实验性标志访问 `astro:assets`，并且这是开始新 Astro 项目的推荐方式。如果你正在迁移到 Astro pre-v3.0，我们建议[启用实验标志来使用资产](/zh-cn/guides/images/)作为图像解决方案。
-:::
-
-将 Gatsby 的 `<StaticImage />` 和 `<GatsbyImage />` 组件与 [Astro 自己的图像集成组件](/zh-cn/guides/images/) 或适当的标准 HTML `<img>` / JSX `<img />` 标签相互转换。
+将 Gatsby 的 `<StaticImage />` 和 `<GatsbyImage />` 组件转换为 [Astro 自己的图像集成组件](/zh-cn/guides/images/#image--astroassets) 或者转化成 React 组件中适当的 [标准 HTML `<img>` / JSX `<img />` 标签](/zh-cn/guides/images/#框架组件中的图像)。
```
Comment (kanweiwei) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-gatsby.mdx:321]:
```suggestion
根据情况在你的 React 组件中将 Gatsby 的 `<StaticImage />` 和 `<GatsbyImage />` 组件转换为 [Astro 自己的图像集成组件](/zh-cn/guides/images/#image--astroassets) 或者转化成 [标准的 HTML `<img>` / JSX `<img />` 标签](/zh-cn/guides/images/#框架组件中的图像)。
```

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-gatsby.mdx [hunk 2]
```diff
@@ -320,41 +318,33 @@ Gatsby 中使用 `gatsby-browser.js` 中的 CSS 导入实现全局样式。在 A
...
+import rocket from '../assets/rocket.png';
 ---
-<Image src={localImage} alt="The Astro Logo" />
-<img src={rocket} alt="A rocketship in space.">
+<Image src={rocket} alt="太空中的火箭飞船。" />
+<img src={rocket.src} alt="太空中的火箭飞船。">
 ```
 
-Astro 的 `<Image />` 和 `<Picture />` 组件是实验性的，并且目前只在 `.astro` 和 `.mdx` 文件中工作。查看这些组件可用的[完整组件属性列表](/zh-cn/guides/images/)，并注意其中几个会与 Gatsby 的属性不同。
+Astro 的 `<Image />` 组件只能在 `.astro` 和 `.mdx` 文件中工作。查看该组件可用的[完整组件属性列表](/zh-cn/guides/images/#属性)，并注意其中几个会与 Gatsby 的属性不同。
```
Comment (kanweiwei) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-gatsby.mdx:332]:
```suggestion
Astro 的 `<Image />` 组件只能在 `.astro` 和 `.mdx` 文件中工作。查看该组件可用的[完整组件属性列表](/zh-cn/guides/images/#属性)，并注意它与 Gatsby 的属性有一些不同。
```

File: src/content/docs/zh-cn/guides/images.mdx [hunk 3]
```diff
@@ -0,0 +1,701 @@
...
+
+### `<Image />` (`astro:assets`)
+
+使用内置的 `<Image />` Astro 组件来显示本地图像和 [配置的远程图像](#授权远程图像) 的优化版本。
+
+`public/` 文件夹中的图像以及未在项目中特别配置的远程图像也可以与 Image 组件一起使用，但不会被处理。
+
+`<Image />` 可以转换本地或授权的远程图像的尺寸、文件类型和质量，以便控制显示的图像。生成的 `<img>` 标签包括 `alt`、`loading` 和 `decoding` 属性，并推断图像尺寸以避免 **累积布局偏移 Cumulative Layout Shift (CLS)**。
+
+:::tip[什么是累积布局偏移(Cumulative Layout Shift)?]
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/images.mdx:57]:
```suggestion
:::tip[什么是累积布局偏移（Cumulative Layout Shift）？]
```

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-gatsby.mdx [hunk 4]
```diff
@@ -320,41 +318,33 @@ Gatsby 中使用 `gatsby-browser.js` 中的 CSS 导入实现全局样式。在 A
...
-<img src={rocket} alt="A rocketship in space.">
+<Image src={rocket} alt="太空中的火箭飞船。" />
+<img src={rocket.src} alt="太空中的火箭飞船。">
 ```
 
-Astro 的 `<Image />` 和 `<Picture />` 组件是实验性的，并且目前只在 `.astro` 和 `.mdx` 文件中工作。查看这些组件可用的[完整组件属性列表](/zh-cn/guides/images/)，并注意其中几个会与 Gatsby 的属性不同。
+Astro 的 `<Image />` 组件只能在 `.astro` 和 `.mdx` 文件中工作。查看该组件可用的[完整组件属性列表](/zh-cn/guides/images/#属性)，并注意它与 Gatsby 的属性有一些不同。
 
-要继续在 Markdown（`.md`）文件中使用标准 Markdown 语法（`![]()`）使用本地图像，请将图像移动到你的 `public/` 文件夹中。你可能需要更新链接到相对 URL。你也可以在这些文件中使用标准的 `<img>` 标签。注意这些 [在 `public/` 中的图像不会被 Astro 优化](/zh-cn/guides/images/)。
+要继续使用标准 Markdown 语法 (`![]()`) [在 Markdown (`.md`) 文件中使用图像](/zh-cn/guides/images/#markdown-文件中的图像)，你可能需要更新链接。 对于本地图像，不支持在 `.md`  文件中直接使用 HTML `<img>` 标签，必须转换为 Markdown 语法。
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-gatsby.mdx:334]:
```suggestion
要继续使用标准 Markdown 语法 (`![]()`) [在 Markdown (`.md`) 文件中使用图像](/zh-cn/guides/images/#markdown-文件中的图像)，你可能需要更新链接。对于本地图像，不支持在 `.md` 文件中直接使用 HTML `<img>` 标签，必须转换为 Markdown 语法。
```

**Review by 100gle — APPROVED** [review]
有几个小地方排版格式稍微修改一下，其他没有什么大问题

File: src/content/docs/zh-cn/guides/images.mdx [hunk 5]
```diff
@@ -0,0 +1,701 @@
...
+
+不是所有用户都能以相同的方式看到图像，因此在使用图像时，无障碍性尤为重要。使用 `alt` 属性为图像提供 [描述性的 alt 文本](https://www.w3.org/WAI/tutorials/images/)。
+
+如果图像仅仅是装饰性的（即不会对页面的理解有所贡献），请设置 `alt=""` 以便屏幕阅读器和其他辅助技术知道忽略该图像。
+
+##### width 和 height (对于 `public/` 和远程图像是必须的)
+
+这些属性定义了要用于图像的尺寸。
+
+当使用本地图像的原始宽高比时，`width` 和 `height` 可以从源文件中自动推断，因此是可选的。
```
Comment (kanweiwei) [inline src/content/docs/zh-cn/guides/images.mdx:141]:
```suggestion
当以原始宽高比使用本地图像时，`width` 和 `height` 可以从源文件中自动推断，并且它们是可选的。
```

File: src/content/docs/zh-cn/guides/images.mdx [hunk 6]
```diff
@@ -0,0 +1,701 @@
...
+##### format
+
+你可以选择指定要使用的 [图像文件类型](https://developer.mozilla.org/zh-CN/docs/Web/Media/Formats/Image_types#common_image_file_types) 输出。
+
+默认情况下，`<Image />` 组件将生成 `.webp` 文件。
+
+##### quality
+
+`quality` 是一个可选属性，可以是：
+- 一个预设值（`low`、`mid`、`high`、`max`），可以在不同格式之间自动归一化。
```
Comment (kanweiwei) [inline src/content/docs/zh-cn/guides/images.mdx:154]:
```suggestion
- 一个预设值（`low`、`mid`、`high`、`max`），可以在不同格式之间自动标准化。
```

**Review by 100gle — APPROVED** [review]
LGTM

File: src/content/docs/zh-cn/guides/images.mdx [hunk 7]
```diff
@@ -0,0 +1,701 @@
...
+
+```astro '"https://example.com/remote-cat.jpg"'
+<img src="https://example.com/remote-cat.jpg" alt="一只睡觉的猫。">
+```
+
+### 选择 `<Image />` 还是 `<img>`
+
+`<Image />` 组件会优化图像，并根据原始宽高比推断宽度和高度（对于本地图像），以避免 CLS。但是，它只适用于某些格式，并且不提供 `<picture>` 元素，也不支持 `srcset`。
+
+当你不能使用 `<Image />` 组件时，使用 HTML `<img>` 元素，例如：
```
Comment (kanweiwei) [inline src/content/docs/zh-cn/guides/images.mdx:259]:
```suggestion
当你不能使用 `<Image />` 组件时，请使用 HTML `<img>` 元素，例如：
```

File: src/content/docs/zh-cn/guides/images.mdx [hunk 8]
```diff
@@ -0,0 +1,701 @@
...
+  - 不支持的图像格式
+  - 当你不想让 Astro 优化你的图像时
+  - 为了在客户端动态访问和更改 `src` 属性
+
+### 授权远程图像
+
+你可以使用 [`image.domains`](/zh-cn/reference/configuration-reference/#imagedomains) 和 [`image.remotePatterns`](/zh-cn/reference/configuration-reference/#imageremotepatterns) 配置图像优化的授权图像源 URL 域和模式列表。这个配置是一个额外的安全层，用于在显示来自外部源的图像时保护你的站点。
```
Comment (kanweiwei) [inline src/content/docs/zh-cn/guides/images.mdx:266]:
```suggestion
你可以使用 [`image.domains`](/zh-cn/reference/configuration-reference/#imagedomains) 和 [`image.remotePatterns`](/zh-cn/reference/configuration-reference/#imageremotepatterns) 配置授权图像源 URL 域和模式列表，以进行图像优化。这个配置是一个额外的安全层，用于在显示来自外部源的图像时保护你的站点。
```

File: src/content/docs/zh-cn/guides/images.mdx [hunk 9]
```diff
@@ -0,0 +1,701 @@
...
+![A starry night sky.](/images/stars.png)
+
+<!-- 其它服务上的远程图像 -->
+<!-- 使用图像的完成 url -->
+![Astro](https://example.com/images/remote-image.png)
+```
+
+本地图像不支持 `<img>` 标签，也不支持 `<Image />` 组件。
+
+如果你需要更多的控制图像属性，我们建议使用 `.mdx` 文件格式，它允许你在 Markdown 语法之外，还包括 Astro 的 `<Image />` 组件或 JSX `<img />` 标签。使用 [MDX 集成](/zh-cn/guides/integrations-guide/mdx/) 为 Astro 添加对 MDX 的支持。
```
Comment (kanweiwei) [inline src/content/docs/zh-cn/guides/images.mdx:322]:
```suggestion
如果你需要对图像属性做更多的控制，我们建议使用 `.mdx` 文件格式，除了 Markdown 语法，你还可以使用 Astro 的 `<Image />` 组件或 JSX `<img />` 标签。使用 [MDX 集成](/zh-cn/guides/integrations-guide/mdx/) 为 Astro 添加对 MDX 的支持。
```

File: src/content/docs/zh-cn/guides/images.mdx [hunk 10]
```diff
@@ -0,0 +1,701 @@
...
+# 我的 Markdown 页面
+
+<!-- 现在可以使用本地图像 -->
+![A starry night sky.](../../images/stars.png)
+
+<!-- 将你的图像放在你的内容旁边！ -->
+![A starry night sky.](./stars.png)
+```
+
+如果你需要更多的控制图像属性，我们建议使用 `.mdx` 文件格式，它允许你在 Markdown 语法之外，还包括 Astro 的 `<Image />` 组件或 JSX `<img />` 标签。使用 [MDX 集成](/zh-cn/guides/integrations-guide/mdx/) 为 Astro 添加对 MDX 的支持。
```
Comment (kanweiwei) [inline src/content/docs/zh-cn/guides/images.mdx:626]:
```suggestion
如果你需要对图像属性做更多的控制，我们建议使用 `.mdx` 文件格式，除了 Markdown 语法，你还可以使用 Astro 的 `<Image />` 组件或 JSX `<img />` 标签。使用 [MDX 集成](/zh-cn/guides/integrations-guide/mdx/) 为 Astro 添加对 MDX 的支持。
```

File: src/content/docs/zh-cn/guides/images.mdx [hunk 11]
```diff
@@ -0,0 +1,701 @@
...
+import rocket from '../images/rocket.svg';
+---
+<img src={rocket} width="250" height="250" alt="太空中的火箭飞船。" />
+
+<img src={rocket.src} width={rocket.width} height={rocket.height} alt="太空中的火箭飞船。" />
+```
+
+### 更新你的 Markdown, MDX, 和 Markdoc 文件
+
+现在可以标准的 Markdown `![alt](src)` 语法，在 Markdown、MDX 和 Markdoc 中引用项目 `src/` 中的相对图像。
```
Comment (kanweiwei) [inline src/content/docs/zh-cn/guides/images.mdx:612]:
```suggestion
现在可以使用标准的 Markdown `![alt](src)` 语法，在 Markdown、MDX 和 Markdoc 中引用项目 `src/` 中的相对图像。
```

**Review by kanweiwei — APPROVED** [review]
LGTM

**Review by sarah11918 — APPROVED** [review]
I see a bunch of LGTM's, so I'm approving, but waiting for the green light from @Yan-Thomas before merging!

Excellent work @liruifengv and thank you @100gle and @kanweiwei !  :raised_hands:

**Pull request merged**